docs/images/pylint.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="155" height="20">
+    <linearGradient id="b" x2="0" y2="100%">
+        <stop offset="0" stop-color="#bbb" stop-opacity=".1"/>
+        <stop offset="1" stop-opacity=".1"/>
+    </linearGradient>
+    <mask id="anybadge_2">
+        <rect width="155" height="20" rx="3" fill="#fff"/>
+    </mask>
+    <g mask="url(#anybadge_2)">
+        <path fill="#555" d="M0 0h107v20H0z"/>
+        <path fill="#4c1" d="M107 0h48v20H107z"/>
+        <path fill="url(#b)" d="M0 0h155v20H0z"/>
+    </g>
+    <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
+        <text x="54.5" y="15" fill="#010101" fill-opacity=".3">pylint</text>
+        <text x="53.5" y="14">pylint</text>
+    </g>
+    <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
+        <text x="132.0" y="15" fill="#010101" fill-opacity=".3">None</text>
+        <text x="131.0" y="14">None</text>
+    </g>
+</svg>
```

Comment:
the formatting of the badge looks a bit off, can you justify the pylint text to be consistent with the other badges?